### PR TITLE
New package: labwc-0.5.0

### LINKED ISSUE #34782: Package Request : labwc wayland compositor
https://github.com/johanmalm/labwc

A Wayland stacking compositor . It is a wlroots-based stacking compositor aiming to be light-weight and independent, with a focus on simply stacking windows well and rendering some window decorations.

### PR DESCRIPTION
**Pull request by notthewave**
#### Testing the changes
- I tested the changes in this PR: **YES**

closes #34782

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

### CODE REVIEW

**Comment by notthewave** [comment]
wayland-scanner is included in wayland-devel but not found in the arm builds. Any idea way?

**Comment by notthewave** [comment]
reformated for 80 char line length

**Comment by paper42** [comment]
> wayland-scanner is included in wayland-devel but not found in the arm builds. Any idea why?

It's a binary that runs on the host, so wayland-devel should be in hostmakedepends.

File: srcpkgs/labwc/template [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'labwc'
+pkgname=labwc
+version=0.5.0
+revision=1
+build_style=meson
+hostmakedepends="pkg-config cmake wayland-devel"
```
Comment (paper42) [inline srcpkgs/labwc/template:6]: Is cmake really required? scdoc is missing for creating a manpage.

File: srcpkgs/labwc/template [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+makedepends="wlroots-devel wayland-protocols libinput-devel cairo-devel
+pango-devel glib-devel libxkbcommon-devel xorg-server-xwayland xcb-util-devel
+libxml2-devel"
+short_desc="Wayland stacking compositor"
+maintainer="notthewave <winklbauer_m@zoho.eu>"
+license="GPL-2.0-only"
+homepage="https://github.com/johanmalm/labwc"
+distfiles="https://github.com/labwc/labwc/archive/refs/tags/${version}.tar.gz"
```
Comment (paper42) [inline srcpkgs/labwc/template:14]:
```suggestion
homepage="https://github.com/labwc/labwc"
distfiles="https://github.com/labwc/labwc/archive/refs/tags/${version}.tar.gz"
```

**Comment by notthewave** [comment]
I don't recall closing this. Might have accidentally messed up.

**Comment by paper42** [comment]
I reopened the PR

* update to 0.5.2
* rebase on top of master

**Comment by paper42** [comment]
continuing in #38082

**Pull request closed without merging**